src/components/Sum.jsx

```diff
@@ -1 +1,12 @@
-// export function Sum() {}
+import React from 'react';
+
+const Sum = ({
```

Comment:
When you use named export, you can export it right before your function
```suggestion
export const Sum = ({
```